### DISCUSSION TITLE
Getting plotman to stop?

### DISCUSSION THREAD

**Discussion by J-Dizzle86** [ericaltendorf/plotman#684] [Q&A]
![image](https://user-images.githubusercontent.com/38279316/119943363-dbdcba00-bf8a-11eb-9e71-6b3a00d45a0c.png)

I've set my max global and max per temp drive plots to 0. I just want to see where the current plots are at, but doing so starts more plots even though it has a max 0? What am I doing wrong?

**Comment by Pakue95** [comment]
Plotman only starts new plots while running as 'plotman plot' or 'plotman interactive'. I think setting the limits to 0 disables them, so when you called 'plotman plot' it just kept creating new processes.
